src/services/entity.js

```diff
@@ -204,4 +204,65 @@ module.exports = class EntityHelper {
 			throw error
 		}
 	}
+
+	/**
+	 * Get list of entity
+	 * @method
+	 * @name list
+	 * @param {Object} query - query params
+	 * @param {String} userId - logged in user id.
+	 * @param {String} searchText - search label in entity.
+	 * @param {Integer} page -  page no.
+	 * @param {Integer} pageSize -  page limit per api.
+	 * @returns {JSON} - Entity search matched response.
+	 */
+	static async list(query, userId, searchText, pageNo, pageSize) {
+		try {
+			let entityType = query.entity_type_id ? query.entity_type_id : ''
+			let filter
+			if (query.read_user_entity == true) {
+				filter = {
+					[Op.or]: [
+						{
+							created_by: '0',
+						},
+						{
+							created_by: userId,
```

Comment:
Will there be ever a case when regular user will have put an entry in entity table that we have to filter by his userId?

Reply:
currently its not require,  in the beginning  user can also create entities in that this filter was added

Reply:
frontend not using this